src/node/consumer/esbuild.ts

```diff
@@ -1,7 +1,15 @@
 import { build } from "esbuild";
 import { readFileSync, rmSync } from "fs";
 
-const { devDependencies } = JSON.parse(readFileSync("./package.json", "utf8"));
+interface DevDependencies {
+    [key: string]: string;
+}
+
+interface PackageJson {
+    devDependencies?: DevDependencies;
+}
+
+const { devDependencies } = JSON.parse(readFileSync("./package.json", "utf8")) as PackageJson;
```

Comment:
Do we even need this anyways? I think we can get rid of the [external](https://esbuild.github.io/api/#external) exclusion list altogether because the dev dependencies aren't (and shouldn't be) imported in the code.

Reply:
I "think" its needed for build, as all the type definitions are used at build time i think - Might be wrong there though - I'll test when back on the computer tomorrow.

Reply:
Tested - you are right - we dont need this. removed

Reply:
Nice, much better that way!